src/block/block_write.c

```diff
@@ -234,7 +234,7 @@ __block_write_off(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_ITEM *buf, wt_of
     /* Buffers should be aligned for writing. */
     if (!F_ISSET(buf, WT_ITEM_ALIGNED)) {
         WT_ASSERT(session, F_ISSET(buf, WT_ITEM_ALIGNED));
-        WT_RET_MSG(session, EINVAL, "direct I/O check: write buffer incorrectly allocated");
+        WT_RET_MSG(session, EINVAL, "Check: write buffer incorrectly allocated");
```

Comment:
From memory, removing direct I/O also removed the need for this alignment, so I suspect this change needs to go a few levels deeper. If suspect we discussed this at some point - do you remember whether we did?

Reply:
Hi @korteland , we did discuss this at some point, and I mistakenly thought we still needed alignment for write_through although that doesn't seem to be the case and we were only aligning for direct_io. I am going to remove the other layers of buffer alignment in a new ticket.

Reply:
For future reference - we spoke offline, and @ruby-oujo created WT-13227. Closing this thread out.